spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java

```diff
@@ -25,6 +23,8 @@
 import org.springframework.transaction.support.TransactionSynchronizationManager;
 import org.springframework.util.Assert;
 
+import com.rabbitmq.client.Channel;
```

Comment:
Looks like you have different import config for AMQP project.
Or am I missing anything here again?
![imports](https://cloud.githubusercontent.com/assets/1071590/11647042/aa3cbcd8-9d31-11e5-9347-f003e10a091b.png)